website/docs/education.md

```diff
@@ -0,0 +1,32 @@
+---
+displayed_sidebar: docs
```

Comment:
What does this do?

Reply:
The name of the sidebar here is confusing (I'll change it), but the function here is to make this page display a sidebar called `docs`, which is defined in the sidebars file and looks like this: 

<img width="191" alt="image" src="https://github.com/risc0/risc0/assets/15272444/98965b37-e57b-469f-8ad5-9af097cfad5a">

<img width="474" alt="image" src="https://github.com/risc0/risc0/assets/15272444/d319c42d-3825-4fe9-8089-3fe97b152cd1">